html/views/overlay/index.js

```diff
@@ -178,12 +188,13 @@ function ensureSkaterExists(skaterId, team) {
 	}
 }
 
-function updateSkaterNumber(me,mb,v) {
-	sv = v;
-	if(v == '' || v == '-' || v == null) { sv = 'ZZZB'; v = '-'; }
-	$('.Number',me).parent().attr('data-sort', sv);
-	$('.Number',mb).parent().attr('data-sort', sv);
-	$(me).parent().sortDivs(); $(mb).parent().sortDivs();
+function updateSort(sel) {
+	var skaterRow = $(sel);
+	var n = $('.Number', sel).text();
+	var sortValue = (skaterRow.hasClass('NoShow') ? '1' : '0')
```

Comment:
This is a bit hard to read, I'd suggest using an If and adding a comment.

Reply:
A comment documenting the compound sort is definitely a good call. I pushed changes and retested.